src/patternfly/components/Check/examples/Check.md

```diff
@@ -60,34 +60,28 @@ cssPrefix: pf-v5-c-check
 
 ### With description
 ```hbs
-{{#> check}}
-  {{#> check-input check-input--attribute='id="check-with-description" name="check-with-description"'}}{{/check-input}}
-  {{#> check-label check-label--attribute='for="check-with-description"'}}Check with description{{/check-label}}
-  {{#> check-description}}
-    Single-tenant cloud service hosted and managed by Red Hat that offers high-availability enterprise-grade clusters in a virtual private cloud on AWS od GCP.
-  {{/check-description}}
+{{#> check check--id="check-with-description" check--HasDescription=true}}
```

Comment:
Is it feasible to update this to

```
{{> check
  check--id="check-with-description"
  check-description--text="Single-tenant cloud service hosted and managed by Red Hat that offers high-availability enterprise-grade clusters in a virtual private cloud on AWS od GCP."
  check-label--text="Check with description"}}
```

and instead of having to specify `check--HasDescription`, just test for `check-description--text`? I have that question for dropping `check--HasBodyText` and testing for `check-body--text`.